packages/nocodb/src/controllers/users/users.controller.ts

```diff
@@ -118,8 +65,9 @@ export class UsersController {
   ])
   @UseGuards(AuthGuard('local'))
   @HttpCode(200)
-  async signin(@Request() req) {
-    return this.usersService.login(req.user);
+  async signin(@Request() req, @Response() res) {
+    await this.setRefreshToken({ req, res });
+    res.json(this.usersService.login(req.user));
```

Comment:
UsersService login function is async and returns a promise (but it is synchronous so not necessary) which we don't await here. This cause empty response to returned by this endpoint, breaking the login process.

So we should drop await from UsersService login function.